tst/EnergyPlus/unit/EvaporativeCoolers.unit.cc

```diff
@@ -433,11 +433,10 @@ TEST_F(EnergyPlusFixture, EvaporativeCoolers_IndEvapCoolerPower)
     OperatingMode DryWetMode(EvaporativeCoolers::OperatingMode::DryFull);
     Real64 FlowRatio(1.0);
 
-    EvapCond(EvapCoolNum).FanPowerModifierCurveIndex = 1;
-    state->dataCurveManager->allocateCurveVector(1);
-    auto *curve = state->dataCurveManager->PerfCurve(1);
+    auto *curve = AddCurve(*state, "Curve1");
```

Comment:
The preferred way of adding curves. `AddCurve` will create the object, add it to the array, and create a map entry. No need to pre-allocate.